app/src/main/java/org/simple/clinic/contactpatient/ContactPatientProfile.kt

```diff
@@ -0,0 +1,62 @@
+package org.simple.clinic.contactpatient
+
+import android.os.Parcelable
+import androidx.room.Embedded
+import androidx.room.Relation
+import kotlinx.parcelize.Parcelize
+import org.simple.clinic.bloodsugar.BloodSugarMeasurement
+import org.simple.clinic.bp.BloodPressureMeasurement
+import org.simple.clinic.facility.Facility
+import org.simple.clinic.medicalhistory.MedicalHistory
+import org.simple.clinic.patient.Patient
+import org.simple.clinic.patient.PatientAddress
+import org.simple.clinic.patient.PatientPhoneNumber
+import org.simple.clinic.patient.businessid.BusinessId
+
+@Parcelize
+data class ContactPatientProfile(
+    @Embedded
+    val patient: Patient,
+
+    @Relation(
+        parentColumn = "addressUuid",
+        entityColumn = "uuid"
+    )
+    val address: PatientAddress,
+
+    @Relation(
+        parentColumn = "uuid",
+        entityColumn = "patientUuid"
+    )
+    val phoneNumbers: List<PatientPhoneNumber>,
+
+    @Relation(
+        parentColumn = "uuid",
+        entityColumn = "patientUuid"
+    )
+    val businessIds: List<BusinessId>,
```

Comment:
I don't think we are using business IDs in the contact patient sheet. Can you verify that and see if we can remove this, please?

Reply:
We're using this in the effectHandler, ptal?

Reply:
We are calling `withoutDeletedBusinessIds` in effect handler. But are we using business IDs from the contact patient profile somewhere? 🤔

Reply:
So if we are not using business IDs we can remove this and the `withoutDeletedBusinessIds` method as well.

Reply:
Yes we'll filter the businessId's received from the ContactPatientProfile in the `withoutDeletedBusinessIds`, right. Hence we cannot remove them.

Reply:
Umm, okay. Why are we filtering them? Are we using it somewhere in the UI, Update or effect handler directly? At this point we are simply filtering them and not really using them right?